### PR TITLE
feat: improved error catching and version bump

### PR DESCRIPTION
**Pull request by Lash-L**
Includes raising HomeAssistantError whenever an command fails to send, bumping the api, and catching an error if the api fails to release

### CODE REVIEW

**Comment by Lash-L** [comment]
This should also hopefully help fix some errors that people are having when a command fails. This way, it will raise an error and Home Assistant knows something went wrong.

File: custom_components/roborock/device.py [hunk 1]
```diff
@@ -102,6 +104,11 @@ def __init__(
...
     async def send(self, command: RoborockCommand, params=None):
         """Send a command to a vacuum cleaner."""
-        response = await self.coordinator.api.send_command(command, params)
+        try:
+            response = await self.coordinator.api.send_command(command, params)
+        except RoborockException as err:
+            raise HomeAssistantError(
+                f"Error while calling {command.name} with {params}"
+            ) from err
         self.coordinator.schedule_refresh()
```
Comment (humbertogontijo) [inline custom_components/roborock/device.py:113]: Make sense to schedule refresh if the command failed?

**Review by humbertogontijo — APPROVED** [review]

**Comment by Lash-L** [comment]
and status and state were swapped

**Comment by humbertogontijo** [comment]
> and status and state were swapped

Good catch

**Comment by humbertogontijo** [comment]
Do you have something else planned to be released or can I make a new release?

**Comment by Lash-L** [comment]
So I would go ahead and put out the release

**Pull request merged**